src/assets/scss/base/_page.scss

```diff
@@ -39,4 +39,8 @@
 				}
 			}
 
-	}
\ No newline at end of file
+	}
+
+	p{

```

Comment:
Take a look at the preview site, you can see this affecting all `p` tags, for example text in the site header that should be center. This needs to be scoped to just the one section that contains the commands.